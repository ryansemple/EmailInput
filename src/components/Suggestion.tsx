import React from "react";

interface ISuggestionProps {
	emailSuggestion: string,
	emailSuggestionClickEvent: any
}

const Suggestions = (props: ISuggestionProps) => (
	<li
		onClick={() => props.emailSuggestionClickEvent(props.emailSuggestion)}
		key={props.emailSuggestion}
		className="Email_Suggestions_Suggestion">
		{props.emailSuggestion}
	</li>
)

export default Suggestions;