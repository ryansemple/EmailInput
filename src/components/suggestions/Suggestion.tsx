import React from "react";

interface ISuggestionProps {
	emailSuggestion: string,
	emailSuggestionClickEvent: any
}

const Suggestion = (props: ISuggestionProps) => (
	<li
		onClick={() => props.emailSuggestionClickEvent(props.emailSuggestion)}
		key={props.emailSuggestion}
		className="Email_Suggestions_Suggestion">
		{props.emailSuggestion}
	</li>
)

export default Suggestion;