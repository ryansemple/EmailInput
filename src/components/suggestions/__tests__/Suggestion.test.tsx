import React from "react";
import { render } from "@testing-library/react";
import Suggestion from "../Suggestion";

test("Renders Suggestion component correctly.", () => {
	const { container } = render
	(
		<Suggestion
			emailSuggestion="suggestion"
			emailSuggestionClickEvent={() => {}}
		/>
	);
	
	expect(container).toMatchSnapshot();
});