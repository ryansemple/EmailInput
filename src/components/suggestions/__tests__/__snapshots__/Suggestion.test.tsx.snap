// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders Suggestion component correctly. 1`] = `
<div>
  <li
    class="Email_Suggestions_Suggestion"
  >
    suggestion
  </li>
</div>
`;
